charts/govuk-rails-app/templates/assets-upload-job.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-uploadRailsAssets
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      containers:
+      - name: {{ .Release.Name }}-uploadRailsAssets
+        image: "{{ required "A valid .Values.appImage.repository entry required!" .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
```

Comment:
We have a warning for the repo but not the image, is it intentional?

Reply:
Not particularly, I lifted this from the dbmigrate job